pages/shared/data/tools.yaml

```diff
@@ -154,9 +154,9 @@ developer:
     description: A collection of AMP tools making it easier to publish and host AMP pages.
     url: https://github.com/ampproject/amp-toolbox
     image:
-      src: https://user-images.githubusercontent.com/380472/36001450-96cfbce2-0d26-11e8-8b65-4ffc3182d57c.gif
-      width: 904
-      height: 680
+      src: /static/img/tools/36001450-96cfbce2-0d26-11e8-8b65-4ffc3182d57c.gif
```

Comment:
Can we not use a GIF here, due to file size. I think a screenshot of the last frame should be fine:

![amp-toolbox-optimizer](https://user-images.githubusercontent.com/380472/57357104-7a164100-7172-11e9-82e3-0a42d4887360.jpg)